src/main/java/com/palantir/gradle/revapi/RevapiAnalyzeTask.java

```diff
@@ -126,16 +131,15 @@ private RevapiConfig revapiIgnores() {
         return RevapiConfig.empty().withIgnoredBreaks(acceptedBreaks.get());
     }
 
-    private API api(SetProperty<File> apiJars, SetProperty<File> dependencyJars) {
+    private API api(Provider<FileCollection> apiJars, Provider<FileCollection> dependencyJars) {
         return API.builder()
                 .addArchives(toFileArchives(apiJars))
                 .addSupportArchives(toFileArchives(dependencyJars))
                 .build();
     }
 
-    private static List<FileArchive> toFileArchives(SetProperty<File> property) {
-        return property.get().stream()
-                .filter(File::isFile)
+    private static List<FileArchive> toFileArchives(Provider<FileCollection> property) {
+        return property.get().filter(File::isFile).getFiles().stream()
```

Comment:
I'd actually just throw here if you get smth that's not a file. We control what goes in here, for the most part, and if smth that's not a file ends up in here, that's bad and we should alert the user that we're not analysing their entire configuration.

Reply:
well, never mind, the `handles the output of extra source sets being added to compile configuration` test promptly told me why we can't just do that.